schemas/app/inventory/StockMovementItem.json

```diff
@@ -46,12 +53,13 @@
       "readOnly": true
     }
   ],
-  "tableFields": ["item", "fromLocation", "toLocation", "quantity", "rate"],
-  "keywordFields": ["item"],
+  "tableFields": ["item", "fromLocation", "toLocation", "batchNumber", "quantity", "rate"],
+  "keywordFields": ["item", "batchNumber"],
```

Comment:
Why is it keywordFields, do you intend to make Stock Movement Items searchable by batch?

Reply:
corrected,  it was unnecessary